jsonschema-module-jackson/pom.xml

```diff
@@ -54,7 +66,20 @@
                 <groupId>org.moditect</groupId>
                 <artifactId>moditect-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-maven-plugin</artifactId>
+                <version>${kotlin.version}</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <jvmTarget>1.8</jvmTarget>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+            </plugin>
```

Comment:
**suggestion**: The Maven compiler plugin is the first one to be mentioned in this `pom.xml`, i.e., this repeated mention is redundant, right?

Reply:
According to Kotlin docs (https://kotlinlang.org/docs/maven.html#compile-kotlin-and-java-sources):
> To compile projects that include Kotlin and Java source code, invoke the Kotlin compiler before the Java compiler. In Maven terms it means that kotlin-maven-plugin should be run before maven-compiler-plugin using the following method, making sure that the kotlin plugin comes before the maven-compiler-plugin in your pom.xml file

That's why the additional `maven-compiler-plugin` was added (automatically by the IDE, BTW). It makes the first mention redundant though - I'll remove it.